db/migrate/20201118180850_add_accountable_to_account.rb

```diff
@@ -0,0 +1,5 @@
+class AddAccountableToAccount < ActiveRecord::Migration[6.0]
+  def change
+    safety_assured { add_reference :accounts, :accountable, polymorphic: true }
```

Comment:
with this change you need to update already existing data
as you changed the association
old company associations need to be rebuilt